boards/arm/mps3_an547/doc/index.rst

```diff
@@ -0,0 +1,191 @@
+.. _mps3_an547_board:
+
+ARM MPS3 AN547
+###############
+
+Overview
+********
+
+The mps3_an547 board configuration is used by Zephyr applications that run
+on the MPS3 AN547 board. It provides support for the MPS3 AN547 ARM Cortex-M55
+CPU and the following devices:
+
+- Nested Vectored Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+- Cortex-M System Design Kit GPIO
+- Cortex-M System Design Kit UART
+- Ethos-U55 NPU
+
+.. image:: img/mps3_an547.png
+     :width: 666px
+     :align: center
+     :height: 546px
+     :alt: ARM MPS3 AN547
+
+More information about the board can be found at the `MPS3 FPGA Website`_.
+
+Hardware
+********
+
+ARM MPS3 AN547 provides the following hardware components:
+
+- ARM Cortex-M55
+- Soft Macro Model (SMM) implementation of SSE-300 subsystem
+- Memory
+
+  - 8MB BRAM
+  - 4GB DDR4 SODIMM (by default, upgradeable to 8GB)
+  - 16GB eMMC
+  - 8MB QSPI Flash
+
+- Debug
+
+  - P‐JTAG, F-JTAG, SWD, 4-bit trace, 16-bit trace
+  - Four serial ports over USB
+
+- Interface
+
+  - AHB GPIO
+  - UART
+  - SPI
+  - I2C
+  - I2S
+  - Color LCD serial interface
+  - Ethernet
+  - VGA
+
+- On-board Peripherals
+
+  - Color LCD
+  - 10 LEDs
+  - 8 Switches
+  - 2 user push buttons
+
+Supported Features
+===================
+
+The mps3_an547 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
```

Comment:
Wonder if here you could list SAU, MPU, etc.

Reply:
We don't seem to do that elsewhere